radio/src/storage/conversions/yaml/yaml_datastructs_zorro.cpp

```diff
@@ -377,7 +377,7 @@ static const struct YamlNode struct_RadioData[] = {
   YAML_SIGNED( "pwrOnSpeed", 3 ),
   YAML_SIGNED( "pwrOffSpeed", 3 ),
   YAML_UNSIGNED( "imperial", 1 ),
-  YAML_UNSIGNED( "jitterFilter", 1 ),
+  YAML_UNSIGNED( "noJitterFilter", 1 ),
```

Comment:
The files in `conversions/yaml/...` shall not be changed anymore, as they generate a format corresponding to EdgeTX 2.6, which is not supposed to change.

Reply:
So I can safely revert the change to those files?

Reply:
I think so, yes.